WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -194,6 +194,29 @@ import WordPressKit
         showGetStarted(from: presenter, jetpackLogin: jetpackLogin, connectedEmail: connectedEmail, siteURL: siteURL)
     }
 
+    /// Used to present the Verify Email flow from the app delegate
+    ///
+    @objc public class func showVerifyEmailForWPCom(from presenter: UIViewController, xmlrpc: String, connectedEmail: String, siteURL: String) {
```

Comment:
Super nit: It would be great to explain a bit about these parameters in the comments.

Reply:
Updated in c99f2a7d5f94f9f6e10ee16ae8879c826c4864f7.